mmv1/third_party/terraform/provider/provider_mmv1_resources.go.erb

```diff
@@ -8,6 +8,7 @@ import (
 	  "github.com/hashicorp/terraform-provider-google/google/services/<%= service -%>"
 	<% end -%>
 
+	"github.com/hashicorp/terraform-provider-google/google/services/apphub"
```

Comment:
As PR https://github.com/GoogleCloudPlatform/magic-modules/pull/10079 has been merged,  `apphub` service should be imported by the line 8. Can you please rebase the main branch? After that, this line could be removed.

Reply:
Done.

Reply:
I wonder if this PR has been rebased from the main branch, or this PR doesn't have the latest revision?

Reply:
Or you can try remove the line 11 first without rebasing to see it works or not.

Reply:
Done.